setup-buildengine.sh

```diff
@@ -360,8 +360,7 @@ Options:
    -c   | --compression <0-9>  compression level of 7z [$OPT_COMPRESSION]
    -nc  | --no-compression     do not create the 7z
    -t   | --tooling <FILE>     tooling tarball <FILE>, must be in current directory
-   -ta  | --target-arm <FILE>  arm target rootstrap <FILE>, must be in current directory
-   -ti  | --target-i486 <FILE> i486 target rootstrap <FILE>, must be in current directory
+   --target-\$arch <FILE>      \$arch target rootstrap <FILE>, must be in current directory
```

Comment:
'$arch' looks weird there. I would use just '<ARCH>' to be consistent with other options. Also, escaping eats one character, so columns are not straight anymore.